tests/frontend/test-class-wpseo-opengraph-image.php

```diff
@@ -127,6 +129,16 @@ public function test_add_image_as_string() {
 		$this->assertEquals( $this->sample_array(), $class_instance->get_images() );
 	}
 
+	/**
+	 * Tests whether has images is false by default.
```

Comment:
Could be clearer, for example: `Tests whether no images are added with empty or missing URL.`